### DISCUSSION TITLE
全局热键win+z等不起作用

### DISCUSSION THREAD

**Discussion by blair2018** [shuax/MouseInc.Settings#78] [General]
win10最新版
全局热键win+n可以用，但是将n换成其他字母键不能正常使用，覆盖不了系统现有的功能。目前在用的是ahk配置的热键。想整合到MouseInc。有时间请您看看能不能修复

**Comment by blair2018** [comment]
怀疑是热键冲突，查资料http://www.xitongcheng.com/jiaocheng/win10_article_50559.html

按修改注册表的方法把ASZ键都禁用了。经过试验z键仍然不起作用。就我自己来说，也就只能改变自己的操作习惯，将z改为a。如果能修复，还请开发者修复提过优先级别。

**Comment by shuax** [comment]
https://support.microsoft.com/en-us/windows/keyboard-shortcuts-in-windows-dcc61a57-8ff0-cffe-9796-cb9706c75eec
这个确实是系统占用了，修复不了。

**Comment by zhjcdw** [comment]
我用的是win11系统，也遇到类似的问题，就是很多系统预设的快捷键，没法在mouseinc当中使用。
我也按照http://www.xitongcheng.com/jiaocheng/win10_article_50559.html  这个方法禁用了部分快捷键，
但是 win+c 这个键没法禁用（注册表禁用无效）。
下载了第三方软件查看，win+c也没有被什么程序占用，但是在mouseinc中指定就是无效。
而且按下win+c 会导致系统资源管理器重启（这个我目前还没有查到是哪个软件调用的）。
我之前用hotkeyp，可以指定win+c，它好像调用级别比系统更高。
因为以前一直用win+c 也比较方便，现在顺手经常按这个组合键，结果经常导致资源管理器重启。

**Comment by shuax** [comment]
后续考虑不使用系统API来做热键，这样就能覆盖其他热键，并且支持不同程序定义不同的快捷键功能。

**Reply by Bush2021** [reply]
太棒了, 感谢耍大
